x/upgrade/client/cli/query.go

```diff
@@ -105,7 +105,7 @@ func GetAppliedPlanCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			return clientCtx.PrintOutput(string(bz))
+			return clientCtx.PrintString(fmt.Sprintf("%s\n", string(bz)))
```

Comment:
ditto

Reply:
This was just printing a string using amino. It doesn't work in proto. So the easiest thing seems to be to just print a string. If a json string is really needed we could use `encoding/json`